source/code/plugin/ApplicationInsightsUtility.rb

```diff
@@ -110,11 +111,17 @@ def initializeUtility()
       end
     end
 
-    def getDockerInfo()
-      dockerInfo = DockerApiClient.dockerInfo
-      if (!dockerInfo.nil? && !dockerInfo.empty?)
-        @@CustomProperties["DockerVersion"] = dockerInfo["Version"]
-        #@@CustomProperties["DockerApiVersion"] = dockerInfo["ApiVersion"]
+    def getContainerRuntimeInfo()
+      containerRuntime = ENV[@@EnvContainerRuntime]
+      if !containerRuntime.nil? && !containerRuntime.empty?
+        @@CustomProperties["ContainerRunTime"] = containerRuntime
+        if containerRuntime.casecmp("docker") == 0
+          dockerInfo = DockerApiClient.dockerInfo
+          if (!dockerInfo.nil? && !dockerInfo.empty?)
+            @@CustomProperties["DockerVersion"] = dockerInfo["Version"]
+            #@@CustomProperties["DockerApiVersion"] = dockerInfo["ApiVersion"]
```

Comment:
Do we need the commented line?

Reply:
> Do we need the commented line?
This line was commented before. not sure why.